interface/wx/uilocale.h

```diff
@@ -189,6 +189,71 @@ class wxUILocale
         conventions.
      */
     bool IsSupported() const;
+
+    /**
+        Adds custom, user-defined language to the database of known languages.
+        This database is used in conjunction with the first form of Init().
+    */
+    static void AddLanguage(const wxLanguageInfo& info);
+
+    /**
+        This function may be used to find the language description structure for the
+        given locale, specified either as a two letter ISO language code (for example,
+        "pt"), a language code followed by the country code ("pt_BR") or a full, human
+        readable, language description ("Portuguese-Brazil").
+
+        Returns the information for the given language or @NULL if this language
+        is unknown. Note that even if the returned pointer is valid, the caller
+        should @e not delete it.
+
+        @see GetLanguageInfo()
+    */
+    static const wxLanguageInfo* FindLanguageInfo(const wxString& locale);
+
+    /**
+        Returns a pointer to wxLanguageInfo structure containing information about
+        the given language or @NULL if this language is unknown. Note that even if
+        the returned pointer is valid, the caller should @e not delete it.
+
+        See AddLanguage() for the wxLanguageInfo description.
+        As with Init(), @c wxLANGUAGE_DEFAULT has the special meaning if passed
+        as an argument to this function and in this case the result of
+        GetSystemLanguage() is used.
+    */
+    static const wxLanguageInfo* GetLanguageInfo(int lang);
+
+    /**
+        Returns English name of the given language or empty string if this
+        language is unknown.
+
+        See GetLanguageInfo() for a remark about special meaning of @c wxLANGUAGE_DEFAULT.
+    */
+    static wxString GetLanguageName(int lang);
+
+    /**
+        Returns canonical name (see GetCanonicalName()) of the given language
+        or empty string if this language is unknown.
+
+        See GetLanguageInfo() for a remark about special meaning of @c wxLANGUAGE_DEFAULT.
+
+        @since 2.9.1
```

Comment:
This is trivial, but I think we should remove it, as this entire class is only available since 3.1.6.
```suggestion
        @since 2.9.1
```

Reply:
Ah, sorry. Just a careless copy&paste issue. You are absolutely right that this line should be removed.